examples/goldilocks_models/find_boundary_utils/compare_boundary.py

```diff
@@ -0,0 +1,164 @@
+import matplotlib.pyplot as plt
+import matplotlib
+import numpy as np
+from mpl_toolkits.mplot3d import Axes3D
+from scipy.linalg import solve
+import torch
+from torch import tensor
+import csv
+import os
+
+robot_option = 1
+file_dir = '/Users/jason-hu/'
+
+# optimization setting
+if robot_option == 0:
+    min_gi1 = -0.325
+    max_gi1 = 0.275
+    min_sl1 = 0.0925
+    max_sl1 = 0.3925
```

Comment:
In the` initial guess.cc`, the scale was decided by the max and min value of the task space. Here I didn't figure out a similar method to decide them instead of hard-coding.